Chaincase.UI/ViewModels/ConnectNodeViewModel.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using Chaincase.Common;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.FeesEstimation;
+using Chaincase.Common.Services;
+using System.Net;
+using WalletWasabi.Exceptions;
+using System.Globalization;
+using System.Threading;
+using NBitcoin.Protocol;
+using NBitcoin.Protocol.Behaviors;
+
+namespace Chaincase.UI.ViewModels
+{
+    public class ConnectNodeViewModel : ReactiveObject
+    {
+        private readonly Global _global;
+        private readonly ChaincaseWalletManager _walletManager;
+        private readonly Config _config;
+        private readonly UiConfig _uiConfig;
+        private readonly FeeProviders _feeProviders;
+        private readonly ChaincaseSynchronizer _synchronizer;
+
+        private string _nodeAddress;
+        private bool _hasSetCustomAddress;
+        private bool? _canConnected;
+
+        public ConnectNodeViewModel(Global global, ChaincaseWalletManager walletManager, Config config, ChaincaseSynchronizer synchronizer)
+        {
+            _global = global;
+            _walletManager = walletManager;
+            _config = config;
+            _synchronizer = synchronizer;
+            _hasSetCustomAddress = _config.GetBitcoinP2pEndPoint().ToString() != "127.0.0.1:8333";
+            if (_hasSetCustomAddress) {
+                _nodeAddress = _config.GetBitcoinP2pEndPoint().ToString();
+                try
+                {
+                    SetNodeAddress();
+                }
+                catch (Exception _) {
+                    _canConnected = false;
+                }
+            }
+        }
+
+        public void SetDefaultNodeAddress() {
+            if (_config.Network == NBitcoin.Network.Main)
+            {
+                _config.SetP2PEndpoint(CreateEndPoint(FormatHostAndPort(_config.DefaultMainNetP2PHost, _config.DefaultMainNetP2PPort)));
```

Comment:
Create a `Config.GetDefaultBitcoinP2pEndPoint()` which uses the constant values to make this method cleaner. 

`_config.SetP2PEndpoint(_config.GetDefaultBitcoinP2pEndPoint()`